client/package.json

```diff
@@ -7,9 +7,17 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "boostrap": "^2.0.0",
+    "pure-react-carousel": "^1.24.1",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.3.0"
+    "react-gesture-gallery": "^1.1.0",
+    "react-gesture-responder": "^2.1.0",
+    "react-responsive-carousel": "^3.1.51",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "3.3.0",
+    "react-slick": "^0.25.2",
+    "slick-carousel": "^1.8.1"
```

Comment:
Can you remove the things you added here? You're going to build this carousel from scratch, not use any pre-built ones

Reply:
done